intel-analytics/BigDL
# python api for PPMLContext

1.init a PPMLConext(with or without parameters)
2.read csv file(plain text or encrypted file)
3.write to a csv file(plain text or encrypted file)

tested on

- python=3.6 & pyspark=2.4.6
- python=3.7 & pyspark=3.1.2

before run simple_query_example.py, make sure

1. run make-dist.sh at /BigDL/scala
2. run prepare_env.sh at /BigDL/python/ppml/dev
to set up the environment

then run `bigdl/ppml/api/simple_query_example.py`  with

python simple_query_example.py \
--simple_app_id 123456 \
--simple_app_key 123456\
--primary_key_path /your/primary/key/path/primaryKey \
--data_key_path /your/data/key/path/dataKey \
--input_path /your/test/data/path \
--output_path /your/output/path \
--input_encrypt_mode AES/CBC/PKCS5Padding or plain_text \
--output_encrypt_mode AES/CBC/PKCS5Padding or plain_text
